### DISCUSSION TITLE
.NET MAUI Push Notifications using Azure Notification Hub. Part 2. Setup .NET MAUI

### DISCUSSION THREAD

**Discussion by giscus** [VladislavAntonyuk/vladislavantonyuk.github.io#91] [Announcements]
# .NET MAUI Push Notifications using Azure Notification Hub. Part 2. Setup .NET MAUI

The article demonstrates how to add push notifications to .NET MAUI application using Azure Notification Hub.

https://vladislavantonyuk.github.io/articles/.NET-MAUI-Push-Notifications-using-Azure-Notification-Hub.-Part-2.-Setup-.NET-MAUI/

**Comment by cissemy** [comment]
About Ios ?

**Comment by marcustindab** [comment]
Thanks for the article. Looks nice, but why not use the nuget package "Microsoft.Azure.NotificationHub"? Are there drawbacks by using that nuget package instead of doing it "manually" with HttpClient?

**Reply by VladislavAntonyuk** [reply]
At the moment of writing the article it didn’t work with MAUI (the latest release of the package was in 2020).
I see Microsoft released a new version 2 months ago, probably it is now compatible with .NET MAUI

**Comment by tommyw1996** [comment]
Really helpful thanks! Managed to eventually get this working after tackling a few issues. Unfortunately we now have to migrate this over to use FCM V1 due to Legacy FCM being out of support from June 2024 ;). Have you attempted at all to migrate your .NET MAUI project to now use FCM V1 notifications with notification hub and firebase messaging? I've not managed to get this working myself yet.
